src/XRP/issued-currency-client.ts

```diff
@@ -192,6 +202,34 @@ export default class IssuedCurrencyClient {
     }
   }
 
+  /**
+   * Subscribes to an account's incoming transactions, and triggers a callback upon recieving each transaction.
+   * @see https://xrpl.org/monitor-incoming-payments-with-websocket.html
+   * @see https://xrpl.org/subscribe.html
+   *
+   * @param account The account from which to subscribe to incoming transactions, encoded as an X-Address.
+   * @param callback The function to trigger upon recieving each transaction from the ledger.
+   * @returns The response from the websocket confirming the subscription.
+   */
+  public async monitorIncomingPayments(
+    account: string,
+    callback: (data: WebSocketTransactionResponse) => void,
+  ): Promise<WebSocketStatusResponse> {
+    const classicAddress = XrpUtils.decodeXAddress(account)
+    if (!classicAddress) {
+      throw XrpError.xAddressRequired
+    }
+    const address = classicAddress.address
```

Comment:
typescript snob nit - it's common these days to do `const {address} = classicAddress` i guess. some of our other repos have this turned on as a lint rule